src/oatpp/parser/json/Utils.cpp

```diff
@@ -203,52 +203,65 @@ oatpp::String Utils::escapeString(p_char8 data, v_buff_size size, bool copyAsOwn
     return String((const char*)data, size, copyAsOwnData);
   }
   auto result = String(escapedSize);
-  v_buff_size i = 0;
   p_char8 resultData = result->getData();
   v_buff_size pos = 0;
-  
-  while (i < safeSize) {
-    v_char8 a = data[i];
-    if(a < 32) {
-      if(a == '\b'){
-        resultData[pos] = '\\'; resultData[pos + 1] = 'b'; pos += 2;
-      } else if(a == '\f'){
-        resultData[pos] = '\\'; resultData[pos + 1] = 'f'; pos += 2;
-      } else if(a == '\n'){
-        resultData[pos] = '\\'; resultData[pos + 1] = 'n'; pos += 2;
-      } else if(a == '\r'){
-        resultData[pos] = '\\'; resultData[pos + 1] = 'r'; pos += 2;
-      } else if(a == '\t'){
-        resultData[pos] = '\\'; resultData[pos + 1] = 't'; pos += 2;
-      } else {
-        resultData[pos] = '\\';
-        resultData[pos + 1] = 'u';
-        oatpp::encoding::Hex::writeUInt16(a, &resultData[pos + 2]);
-        pos += 6;
+
+  {
```

Comment:
Just curious, why an additional block is used here?

Reply:
In order for `i` variable not to conflict with a next loop `for(v_buff_size i = pos; i < result->getSize(); i++) {`. Without the block, there was a warning about `i` variable hiding the definition from the outer scope.